squire_sdk/src/server/gathering/hall.rs

```diff
@@ -85,7 +94,9 @@ where
         match msg {
             GatheringHallMessage::NewGathering(id) => self.process_new_gathering(id).await,
             GatheringHallMessage::NewConnection(id, user, ws) => {
-                self.process_new_onlooker(id, user, ws).await
+                if !self.process_new_onlooker(id, user, ws).await {
+                    panic!("TODO what if this fails")
```

Comment:
From another thread:

> The only time a message can fail is when a permanent actor has panicked.

Okay that's something I hadn't internalized. I see that `track` waits for a response, and will panic if the actor is permanent and panics (not sure about transient actors that panic) whereas `send` fires and forgets, ignoring(?) actor panics.

With that in mind, it seems that this panic would be ignored?

```
        self.gatherings
            .send(GatheringHallMessage::NewConnection(id, user, ws))
```

As well as the panics below in `GatheringHallMessage::Persist` since they're scheduled with a delay:

```
    scheduler.schedule(
        Instant::now() + Duration::from_secs(5),
        GatheringHallMessage::Persist,
    );
```

Are these actually okay to ignore?